### DISCUSSION TITLE
Theurgy (a mod for automating basic materials)

### DISCUSSION THREAD

**Discussion by AbsorbingSafe11** [NillerMedDild/Enigmatica6#1778] [Ideas]
Theurgy is a mod by the Occultism dev, which was for the balanced conversion of one material into another. (NOT PROJECTE)
(https://www.curseforge.com/minecraft/mc-mods/theurgy)

I hope it could be added to the modpack, as generating basic Minecraft materials, such as feathers, ink sacs, and to a lesser extent leather are time consuming and/or reliant on only vanilla means. (like a chicken or cow farm)

With Theurgy, players would be able to turn excess materials into Essentia and then into the necessary materials as well as farm crystals to grow Essentia. (more information is within the Theurgy Guidebook)

Sorry if I posted this in the wrong location or if this mod was already proposed. (this is my first modpack suggestion on GitHub)

**Comment by NielsPilgaard** [comment]
Welcome to GitHub 😄 normally mod suggestions are made as issues, but I suppose here is just fine too. We'll think about adding Theurgy :)

**Reply by pm065** [reply]
fifteen centuries later... :smile:
In all seriousness, Theurgy is great but it'd take ages to set up...
